util/import.ts

```diff
@@ -1,11 +1,56 @@
 import { TestCaseInfo } from '../state/atoms/patientTestCase';
 
-export function bundleToTestCase(bundle: fhir4.Bundle): TestCaseInfo {
+/**
```

Comment:
I feel like I'm being a bit persnickety, but would you mind adding the jsdoc style @param and @returns bits? A part of my motivation was that when I first looked at it, I couldn't tell whether populationGroupCodes was the codes that were desired for the test case or the codes that were relevant to the measure. I figured it out, but I'm inclined toward making it as easy as possible. I know that style guide suggests "Avoid uses of typed JSDoc parameters and returns when possible", but I think this is a case where it's beneficial (and chatted with Matt about where it makes sense to draw that line, so style guide may get a couple of bullets updated for that section).